practise_of_ternary_operator.c

```diff
@@ -0,0 +1,9 @@
+#include<stdio.h>
+int main()
+{
+      int a,b;
+      printf("enter values of a and b");
+      scanf("%d %d",&a,&b);
+      (a%b==0)?(printf("%d is multiple of %d",a,b)):(printf("%d is not multiple of %d",a,b));
```

Comment:
Could you please add a multiline comment explaining this line?

Reply:
The user is allowed to enter values of a and b by the use of scanf() function.Modulus operator (%)gives the remainder.I have used "Ternary Operator".Therefore if the condition(a%b==0)is correct program will execute 1st print statement if the condition founds to be incorrect program will execute 2nd print statement.